tests/unit/db/services/test_user_token_service.py

```
@@ -7,49 +7,38 @@
-from src.main_app.db.services.users_service import create_user
+from src.main_app.db.services.user_token_service import UserTokenService
+from src.main_app.db.services.users_service import UsersService
 
 
 def test_delete_user_cascades(mock_app: Flask) -> None:
     with mock_app.app_context():
-        user = create_user("svc_dave")
-        upsert_user_token(user_id=user.user_id, access_key="k", access_secret="s")
-        assert get_user_token(user.user_id) is not None
+        user = UsersService().create_user("svc_dave")
+        UserTokenService().upsert_user_token(user_id=user.user_id, access_key="k", access_secret="s")
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "access_secret"

(S106)